lua/bootstrap.lua

```diff
@@ -0,0 +1,48 @@
+local M = {}
+
+local function join_paths(...)
+  local path_sep = vim.loop.os_uname().version:match "Windows" and "\\" or "/"
+  local result = table.concat(vim.tbl_flatten { ... }, path_sep):gsub(path_sep .. "+", path_sep)
+  return result
+end
+
+function M:init()
+  local default_config_dir = vim.fn.stdpath "config"
+  local default_runtime_dir = vim.fn.stdpath "data"
+  local default_cache_dir = vim.fn.stdpath "cache"
+
+  DATA_PATH = default_runtime_dir
+  CACHE_PATH = default_cache_dir
+  TERMINAL = vim.fn.expand "$TERMINAL"
+  USER = vim.fn.expand "$USER"
+
+  -- Always use the default runtimepath on Windows
+  if not os.getenv "LUNARVIM_RUNTIME_DIR" then
+    LUNARVIM_CONFIG_DIR = default_config_dir
+    LUNARVIM_RUNTIME_DIR = join_paths(default_runtime_dir, "site")
+    LUNARVIM_PACK_DIR = join_paths(default_runtime_dir, "site", "pack")
+    return
+  end
+
+  LUNARVIM_CONFIG_DIR = os.getenv "LUNARVIM_CONFIG_DIR"
+  LUNARVIM_RUNTIME_DIR = os.getenv "LUNARVIM_RUNTIME_DIR"
+  LUNARVIM_PACK_DIR = join_paths(LUNARVIM_RUNTIME_DIR, "site", "pack")
+
+  vim.opt.rtp:remove(default_config_dir)
+  vim.opt.rtp:remove(join_paths(default_config_dir, "after"))
+  vim.opt.rtp:append(LUNARVIM_CONFIG_DIR)
+  vim.opt.rtp:append(join_paths(LUNARVIM_CONFIG_DIR, "after"))
+
+  vim.opt.rtp:remove(join_paths(default_runtime_dir, "site"))
+  vim.opt.rtp:remove(join_paths(default_runtime_dir, "site", "after"))
+
+  vim.opt.rtp:append(join_paths(LUNARVIM_RUNTIME_DIR, "site"))
+  vim.opt.rtp:append(join_paths(LUNARVIM_RUNTIME_DIR, "site", "after"))
+
+  -- TODO: we need something like this: vim.opt.packpath = vim.opt.rtp
+  vim.cmd [[let &packpath = &runtimepath]]
+
+  vim.cmd("set spellfile=" .. require("utils").join_paths(LUNARVIM_CONFIG_DIR, "spell", "en.utf-8.add"))
```

Comment:
What is this for ?

Reply:
I have no idea but it broke neovim when I didn't have it set. It was already defined from before, just shorter and more hard-coded 😃

Reply:
possibly related to https://github.com/LunarVim/LunarVim/issues/1445